greenmail-webapp/src/main/webapp/WEB-INF/web.xml

```diff
@@ -14,15 +14,15 @@
     </context-param>
     <context-param>
         <param-name>greenmail.smtp.port</param-name>
-        <param-value>10025</param-value>
+        <param-value>1025</param-value>
```

Comment:
this is backward incompatible change, a main blocker for eventual merging back the change to original repo.  Can you take a look if you couldn't override those value with context.xml? Then you would have context.xml (for example context-imap.xml that you could rename to context.xml if needed on deployment)